src/main/java/seedu/address/logic/parser/AddPolicyCommandParser.java

```diff
@@ -57,7 +58,7 @@ public AddPolicyCommand parse(String args) throws ParseException {
         ExpiryDate expiryDate = ParserUtil.parseExpiryDate(
                 argMultimap.getValue(PREFIX_POLICY_EXPIRY_DATE).orElse(""));
 
-        Policy policy = Policy.makePolicy(policyType, premiumAmount, coverageAmount, expiryDate);
+        Policy policy = Policy.makePolicy(policyType, premiumAmount, coverageAmount, expiryDate, new ClaimList());
```

Comment:
use null for claimList. The respective constructors for LifePolicy, HealthPolicy, and EducationPolicy will check for null inputs and create the new empty ClaimList. Do the same for all instance where you called makePolicy with an empty ClaimList.